content/course-specific-material/introR-Remote-Apr20.Rmd

```diff
@@ -0,0 +1,77 @@
+---
+title: "Remote Intro to R Training 2020"
+date: "2020-04-01"
+slug: "introR-Remote-Apr20"
+output: USGSmarkdowntemplates::hugoTraining_course
+parent: Course Specific Material
+---
+
+April 7 -9, 2020 remotely via WebEx
+
+### Installation
+
+See [R Installation Instructions](/installr) for information on what software should be installed prior to the course. 
+
+### Tentative schedule
+
+Times are in Eastern time zone.
+
+**Day 1**
+
+* 10:00 am - 10:30 am – Instructors available for questions
+* 10:30 am - 11:30 am – [Introduction](/intro-curriculum/Introduction)
+* 11:30 am - 12:00 pm – [Example workflow](/intro-curriculum/data/data_workflow.Rmd)
+* 12:00 pm - 12:15 pm – Break
+* 12:15 pm - 01:30 pm – [Get](/intro-curriculum/Get)
+* 01:30 pm - 02:00 pm – Lunch
+* 02:00 pm - 02:30 pm – Finish Get
+* 02:30 pm - 03:30 pm – [Clean](/intro-curriculum/Clean)
+* 03:30 pm - 03:45 pm – Break
+* 03:45 pm - 05:00 pm – More Clean
+
+**Day 2**
+
+* 10:00 am - 10:30 am – Instructors available for questions
+* 10:30 am - 11:30 am – Finish Clean & do "Clean" startup exercise
+* 11:30 am - 12:00 pm – [Explore](/intro-curriculum/Explore)
+* 12:00 pm - 12:15 pm – Break
+* 12:15 pm - 01:30 pm – Finish explore
+* 01:30 pm - 02:00 pm - Lunch
+* 02:00 pm - 03:30 pm – [Analyze: Base](/intro-curriculum/Analyze)
+* 03:30 pm - 03:45 pm – Break
+* 03:45 pm - 05:00 pm – [Analyze: dataRetrieval](https://cran.r-project.org/web/packages/dataRetrieval/dataRetrieval.pdf) 
+
+**Day 3**
+
+* 10:00 am - 10:30 am – Instructors available for questions
+* 10:30 am - 12:00 pm – Visualize with ggplot2: [Visualize](/intro-curriculum/ggplot2/)
+* 12:00 pm - 12:15 pm – Break
+* 12:15 pm - 01:30 pm – [Repeat](/intro-curriculum/Reproduce/)
+* 01:30 pm - 02:00 pm - Lunch
+* 02:00 pm - 03:30 pm – Finish Repeat
+* 03:30 pm - 03:45 pm – Break
+* 03:45 pm - 04:30 pm – Project time: [USGS R packages](/intro-curriculum/USGS/), projects (group/individual), [Beyond Basic R blog topics](https://waterdata.usgs.gov/blog/tags/beyond-basic-r/), or [additional topics](/intro-curriculum/Additional/)
+
+### Data files
```

Comment:
Let's delete this data section since we send via the zip file now.

Reply:
Just added this in my new commit